docs/examples/getting_started.txt

```diff
@@ -1,4 +1,4 @@
-python -m marbles docs/examples/getting_started.py
+$ python -m marbles docs/examples/getting_started.py
```

Comment:
Do this in the readme as well?

Reply:
The other examples in the README don't have a bash prompt